src/manager.rs

```diff
@@ -228,37 +241,43 @@ impl Manager {
 
     pub fn enable_poll_with_intensity(&mut self, value: &ValueID, intensity: u8) -> bool {
         unsafe {
-            extern_manager::manager_enable_poll_with_intensity(self.ptr, value.ptr, intensity)
+            let cvid = extern_value_id::value_id_from_packed_id(value.home_id, value.id);
```

Comment:
for this PR or for later, as you wish, but as I suggested, it would be clearer to have a shorter ExternValueID constructor.